Project/Source/Resources/ResourceTexture.cpp

```diff
@@ -54,4 +55,8 @@ void ResourceTexture::Load() {
 
 void ResourceTexture::Unload() {
 	glDeleteTextures(1, &glTexture);
+}
+
+void ResourceTexture::OnEditorUpdate() {
+	ImGui::Text("Test Working");
```

Comment:
It will be changed in other PR

Reply:
Add a TODO to make sure it gets taken care of later.